### PR TITLE
 fix underline at bottom of navbar #2

### PR DESCRIPTION
**Pull request by OMIIIEE**
image
fixed and made underline at the bottom of the navbar and also fixed underline behavior in responsive page 
image

### CODE REVIEW

**Comment by BorisEdison** [comment]
> image fixed and made underline at the bottom of the navbar and also fixed underline behavior in responsive page image

image
#### its looking like this, 

image
#### 20px suits better, check and let me know

#### and please shift the js to this file

https://github.com/pandevs-org/rumblesheetJS-website/blob/main/public/js/index.js

File: public/styles/navbar.css [hunk 1]
```diff
@@ -78,7 +80,7 @@ nav>.container-fluid>#navbarNavAltMarkup {
 
 .nav-link:before {
 	position: absolute;
-	bottom: 0px;
+	bottom: -24px;
```
Comment (BorisEdison) [inline public/styles/navbar.css:83]: 20px suits better it seems, but make sure to keep the navbar height to at max 75px 
image

File: index.html [hunk 2]
```diff
@@ -137,6 +137,20 @@ <h2 class="home-info mb-0 text-center">Seamlessly integrate dynamic, interactive
 
     <!-- local js -->
     <script src="./public/js/index.js"></script>
+<script>
```
Comment (BorisEdison) [inline index.html:140]: shift this code to js file 
https://github.com/pandevs-org/rumblesheetJS-website/blob/main/public/js/index.js

File: views/docs/latest/getting-started/getting-started.html [hunk 3]
```diff
@@ -206,6 +208,21 @@
     <script src="../../public/js/sidebar.js"></script>
     <script src="../../../../public/js/index.js"></script>
 
+    <script>
```
Comment (BorisEdison) [inline views/docs/latest/getting-started/getting-started.html:211]: shift this code to js file
https://github.com/pandevs-org/rumblesheetJS-website/blob/main/public/js/index.js

**Comment by OMIIIEE** [comment]
yes sure ! I'll Make the changes @BorisEdison

**Comment by OMIIIEE** [comment]
please review the changes and let me know to change anything . after setting max height to 75px it looks like this , but I have to set margin to 18px as for 20px it goes out of navbar in my browser 
image
 
image

File: index.html (hunk 2 above)
Comment (BorisEdison) [reply index.html:140]: remove this code here, as it is already present in index.js

File: public/styles/navbar.css [hunk 4]
```diff
@@ -45,11 +47,21 @@ nav>.container-fluid>#navbarNavAltMarkup {
...
+	max-height: 75px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	align-content: center;
+
 }
 
 .navbar-unscrolled {
-	padding: 24px 48px 8px;
```
Comment (BorisEdison) [inline public/styles/navbar.css:52]: for navbar-unscrolled its not required that the navbar is restricted to 75px max and it also need to be vertically centered,

you can check the behaviour I want in https://borisedison.in for reference

**Review by BorisEdison — CHANGES_REQUESTED** [review]

**Review by BorisEdison — APPROVED** [review]

**Pull request merged**